build.sh

```diff
@@ -160,11 +160,16 @@ build_combined() {
     # Combine ar archives
     LIPO_OUTPUT="$out_path/$product_name/$module_name"
     xcrun lipo -create "$simulator_path/$binary_path" "$os_path/$binary_path" -output "$LIPO_OUTPUT"
-    if [[ "$destination" != "" ]] && [[ "$config" == "Release" ]]; then
+
+    if [[ $(xcode_major_version) != "6" && "$destination" != "" && "$config" == "Release" ]]; then
```

Comment:
I thought about that, but it didn't feel right to me since the relationship between bitcode and the Swift version isn't as clear as the relationship between bitcode and the Xcode version. If you think it's simpler, I can switch it though.